eng/pipelines/templates/steps/wrapped-use-python.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
```

Comment:
nit: I'd call this template just `use-python-version.yml`

Reply:
👍